app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, vstr: dict) -> str:
+        if "vaccine" not in vstr.keys() or vstr["vaccine"] == None:
+            raise NotVaccinatedError("not vaccinated")
+        elif vstr["vaccine"]["expiration_date"] < datetime.date.today():
```

Comment:
1. I would use if instead of elif - it increases flexibility and reduces nesting. If raise is executed, the code does not proceed further.
2. Using < condition, we exclude today's date. On the expiration date, the vaccine is still valid